lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/TitleBarReactViewController.java

```diff
@@ -44,6 +44,11 @@ public void sendOnNavigationButtonPressed(String buttonId) {
 
     }
 
+    @Override
+    public String getCurrentComponentName() {
```

Comment:
It was complaining that `getCurrentComponetName` needs to be implemented. Should this return `null` instead?

Reply:
Yes

Reply:
Now returning:

```java
@Override
public String getCurrentComponentName() {
  return null;
}
```